bftengine/src/bftengine/ViewsManager.cpp

```diff
@@ -272,6 +272,42 @@ NewViewMsg* ViewsManager::getMyNewViewMsgForCurrentView() {
   return r;
 }
 
+vector<ViewChangeMsg*> ViewsManager::getViewChangeMsgsForCurrentView()
```

Comment:
Should probably return a vector of unique_ptrs so the caller doesn't have to free them.